lib/Cleantalk/ApbctWP/Firewall/AntiCrawler.php

```diff
@@ -259,7 +260,8 @@ public function check()
                     Cookie::get('wordpress_apbct_antibot') !== hash(
                         'sha256',
                         $this->api_key . $this->apbct->data['salt']
-                    )
+                    ) &&
+                    Cookie::get('apbct_bot_detector_exist')
```

Comment:
If Anti-Crawler enabled, but Bot-Detector disabled, screen of Anti-Crawler work as expected?

Reply:
I will make an amendment to the working conditions, it should be: if there is a cookie from bot-detector, then another cookie from AntiCrawler is not required and vice versa. When the bot-detector is turned off, the AntiCrawler cookie will be checked, when the bot-detector turned on check cookie is bot-detector  || AntiCrawler